cli/scancommands.go

```diff
@@ -196,7 +196,10 @@ func EnrichCmd(c *components.Context) error {
 func ScanCmd(c *components.Context) error {
 	if len(c.Arguments) == 0 && !c.IsFlagSet(flags.SpecFlag) {
 		return pluginsCommon.PrintHelpAndReturnError("providing either a <source pattern> argument or the 'spec' option is mandatory", c)
+	} else if len(c.Arguments) > 1 {
+		log.Warn(utils.GetCliTooManyArgsErrorMessage(len(c.Arguments), c.CommandName))
 	}
```

Comment:
```suggestion
	} else if len(c.Arguments) > 1 {
		log.Warn(utils.GetCliTooManyArgsErrorMessage(len(c.Arguments), c.CommandName))
		return pluginsCommon.PrintHelpAndReturnError(c)
	}
```
If you think we should stop the command instead of just logging you can use the above (and remove the last part of the log about running the --help since it will show the output)